src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -362,7 +362,8 @@ protected boolean canBeInstantiated() {
 				.description("A location in a <a href='#world'>world</a>. Locations are world-specific and even store a <a href='#direction'>direction</a>, " +
 						"e.g. if you save a location and later teleport to it you will face the exact same direction you did when you saved the location.")
 				.usage("")
-				.examples("")
+				.examples("teleport player to location at 0, 69, 0",
+						  "teleport player to location at 0, 69, 0 in world 'world'")
```

Comment:
```suggestion
						  "teleport player to location at 0, 69, 0 in world \"world\"")
```
Skript doesn't use single quotes

Reply:
Good catch. Thanks.

Reply:
I meant the example in a way to replace 'world' with something... But examples should be executable. Sorry, I'm new to Skript. A few days ago, I was struggling with starting up the server. :)